app/assets/stylesheets/application.scss

```diff
@@ -9,7 +9,7 @@
  * compiled file so the styles you add here take precedence over styles defined in any other CSS/SCSS
  * files in this directory. Styles in this file should be added after the last require_* statement.
  * It is generally better to create a new file per style scope.
- *
- *= require_tree .
- *= require_self
  */
```

Comment:
sao lại xóa 2 dòng require đi?

Reply:
Do not use *= require in Sass or your other stylesheets will not be able to access the Bootstrap mixins or variables.
If you want to use multiple Sass files, you should generally use the Sass @import rule instead of these Sprockets directives. When using Sprockets directives, Sass files exist within their own scope, making variables or mixins only available within the document they were defined in.

Reply:
Do em import bootstrap qua application file, còn import boostrap qua custom file thì cũng ko cần xóa